### PR TITLE
First C++ Program

### PR DESCRIPTION
**Pull request by Lal2023**
Adding default C++ source code

### CODE REVIEW

File: program.cpp [hunk 1]
```diff
@@ -0,0 +1,8 @@
+#include <iostream>
+
+using namespace std;
+
+int main(){
+    cout << "This is the configuration file" << endl;
+    cout << "Version 1.0" << endl;
```
Comment (Rowdy2018) [inline program.cpp:7]: It doesn't hurt to show version of the program here.

Comment (Lal2023) [reply program.cpp:7]: Looks good

**Review by Rowdy2018 — APPROVED** [review]
Looks like everything is working

**Comment by Rowdy2018** [comment]
Closed

**Pull request merged**